src/CheckIfDead.php

```diff
@@ -467,6 +467,7 @@ public function sanitizeURL( $url, $stripFragment = false, $preserveQueryEncodin
 		// This avoids possible 400 Bad Response errors.
 		$url .= "/";
 		if ( isset( $parts['path'] ) && strlen( $parts['path'] ) > 1 ) {
+			$parts['path'] = str_replace( "+", "CHECKIFDEADPLUSSPACE", $parts['path'] );
```

Comment:
Can you add a comment explaining why we need to worry about pluses in the path (and possibly add an example URL)?